lib/client.js

```diff
@@ -220,6 +220,22 @@ function parseHeader(str) {
 }
 
 /**
+ * Check if `mime` is json or has +json structured syntax suffix.
+ *
+ * @param {String} mime
+ * @return {Boolean}
+ * @api private
+ */
+
+function isJSON(mime) {
+  if (!mime) return false;
+  var parts = mime.split('/');
+  var type = parts[0];
+  var subtype = parts[1];
+  return subtype && subtype.match(/json/i);
```

Comment:
The collect is not `/[\\+]json\b/.test(mime)` but `/[\/+]json\b/.test(mime)`, I suppose. And, why `\b` (not`$`)?